File: app/src/main/java/ua/edu/deanoffice/mobile/studentchdtu/activities/ExamApplicationActivity.java
package ua.edu.deanoffice.mobile.studentchdtu.activities;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Calendar;

import ua.edu.deanoffice.mobile.studentchdtu.mobile.ApplicationCache;
import ua.edu.deanoffice.mobile.studentchdtu.R;
import ua.edu.deanoffice.mobile.studentchdtu.mobile.Mobile;

public class ExamApplicationActivity extends AppCompatActivity {

    private int id = 7;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_exam_application);
        Log.d("Test", "Enter");

        TextView applicationHeader = findViewById(R.id.applicationHeader);
        TextView applicationBody = findViewById(R.id.applicationBody);
        TextView applicationDateTime = findViewById(R.id.applicationDateTime);

        applicationHeader.setText(Mobile.getInstance().getCurrentApplication().getHeader());
        applicationBody.setText(Mobile.getInstance().getCurrentApplication().getBody());
        DateFormat df = new SimpleDateFormat("dd.MM.yyyy");
        String date = df.format(Calendar.getInstance().getTime());
        applicationDateTime.setText(date);
    }

    @Override
    public void onBackPressed() {
        Intent intent = new Intent(ExamApplicationActivity.this, ChooseApplicationActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/ua/edu/chdtu/deanoffice/mobile/myapplication/ExamApplicationActivity.kt
package ua.edu.chdtu.deanoffice.mobile.myapplication

import android.icu.util.Calendar
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import java.text.DateFormat
import java.text.SimpleDateFormat

class ExamApplicationActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_exam_application)

        var applicationHeader = findViewById<TextView>(R.id.applicationHeader)
        var applicationBody = findViewById<TextView>(R.id.applicationBody)
        var applicationDateTime = findViewById<TextView>(R.id.applicationDateTime)

        applicationHeader.setText(ApplicationDataTemp.header.replace("\\n", "\n"))
        applicationBody.setText(ApplicationDataTemp.body)
        val df: DateFormat = SimpleDateFormat("dd.MM.yyyy")
        val date: String = df.format(Calendar.getInstance().time)
        applicationDateTime.setText(date)
    }
}
